ecomstore/tests/backend/products/test_create_products_smoke.py

```diff
@@ -0,0 +1,36 @@
+
+import pytest
+from ecomstore.src.utilities.genericUtility import generate_random_string
+from ecomstore.src.api_helpers.ProductsAPIHelper import ProductsAPIHelper
+from ecomstore.src.dao.products_deo import ProductsDao
+
+pytestmark = [pytest.mark.products, pytest.mark.smoke, pytest.mark.BE]
```

Comment:
everytest should have 'regression' tag. All tests should run as regression but only few test will run as 'smoke'.